packages/storybook/src/nimble/drawer/drawer.stories.ts

```diff
@@ -149,12 +150,15 @@ const metadata: Meta<DrawerArgs> = {
     argTypes: {
         location: {
             options: [DrawerLocation.left, DrawerLocation.right],
-            control: { type: 'radio' }
+            control: { type: 'radio' },
+            description: 'The side of the screen from which the drawer will slide in',
+            table: { category: apiCategory.attributes }
         },
         preventDismiss: {
             name: 'prevent-dismiss',
             description:
-                'A boolean attribute to configure whether or not the drawer is dismissible via the `Esc` key, or any other dismiss action that is supported in the future'
+                'A boolean attribute to configure whether or not the drawer is dismissible via the `Esc` key, or any other dismiss action that is supported in the future.',
```

Comment:
We shouldn't say "A boolean attribute...", and it's probably not necessary to talk about hypothetical, future, dismiss actions.

Note that we should also be consistent about how we refer to keyboard keys. In the dialog's `show()` documentation, we use "ESC", but here (and below in the `show()` doc) we use "`Esc`". In the tooltip docs we use "Escape". I didn't search for other keys.
```suggestion
                'Prevents the `Esc` key from closing the drawer.',
```